### ISSUE TITLE
How can I finetune keras-facenet?

### ISSUE THREAD

**Issue by ZER-0-NE** [nyoki-mtl/keras-facenet#7]
I want to finetune keras-facenet as was described with VGG16 model [Fine-tuning the top layers of a a pre-trained network](https://blog.keras.io/building-powerful-image-classification-models-using-very-little-data.html).

How can I do so? How should I define the classifier on top of this model?

**Comment by jadhavsaurabh** [comment]
I have same issue but want to fine tune using triplet loss

**Comment by jadhavsaurabh** [comment]
@ZER-0-NE You got the solution?

**Comment by ZER-0-NE** [comment]
No. I used keras-vggface instead.

**Comment by kevinisbest** [comment]
@ZER-0-NE Hi, are you using this [keras-vggface](https://github.com/rcmalli/keras-vggface)?
Did you compare the accuracy and speed with this repo?

**Comment by ZER-0-NE** [comment]
No

**Comment by x3393467** [comment]
@jadhavsaurabh I want to fine turn too. did you find the right way?
